DesktopUI2/DesktopUI2/DesktopUI2.csproj

```diff
@@ -395,4 +402,9 @@
 			<PrivateAssets>all</PrivateAssets>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="navisworks.gui.common">
+      <HintPath>..\..\..\..\..\..\..\..\Program Files\Autodesk\Navisworks Manage 2023\navisworks.gui.common.dll</HintPath>
```

Comment:
Who is this guy? :) Can we remove it?
Will break the CI if committed.

Reply:
Done